contracts/SnappAuction.sol

```diff
@@ -56,9 +75,76 @@ contract SnappAuction is SnappBase {
         return auctions[slot].appliedAccountStateIndex != 0;
     }
 
+    function getStandingOrderHash(uint16 userId, uint128 nonce) public view returns (bytes32) {
```

Comment:
Do we need all these getters? Or is this just expected for the future?

Reply:
It is expected for the future and I want to add some more tests